.github/workflows/lint-autoformat.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       contents: read
       pull-requests: write
     runs-on: lf.linux.2xlarge
-    if: ${{ github.repository_owner == 'pytorch' }}
+    if: ${{ github.repository_owner == 'pytorch' && github.event.pull_request.user.login != 'ezyang' }}
```

Comment:
Please... 
```suggestion
    if: ${{ github.repository_owner == 'pytorch' && github.event.pull_request.user.login != 'ezyang' && github.event.pull_request.user.login != 'malfet' }}
```

Reply:
Can we just not run them on draft PRs?

Reply:
May be the right solution is just to disable the workflow or make it an opt-in. Or perhaps run it only if your workflows needs approval...

Reply:
sure

Reply:
a sensible opt-in strategy is have the lint error ci message say "to suggest these as changes on your PR, comment BLAH" and then the flow triggers on that